src/components/isLoading/Loading.tsx

```diff
@@ -0,0 +1,12 @@
+import './Loading.scss';
+
+export const Loading = () => {
```

Comment:
It's better o name your component the same as folder one. Rename to Loader or Preloader